functions/__fzf_search_git_log.fish

```diff
@@ -10,12 +10,12 @@ function __fzf_search_git_log --description "Search the git log of the current g
             # see documentation for git format placeholders at https://git-scm.com/docs/git-log#Documentation/git-log.txt-emnem
             # %h gives you the abbreviated commit hash, which is useful for saving screen space, but we will have to expand it later below
             git log --color=always --format=format:'%C(bold blue)%h%C(reset) - %C(cyan)%as%C(reset) %C(yellow)%d%C(reset) %C(normal)%s%C(reset)  %C(dim normal)[%an]%C(reset)' | \
-            fzf --ansi --tiebreak=index --preview='git show --color=always (string split --max 1 " " {})[1]'
+            fzf --ansi --tiebreak=index --preview='git show --color=always (string split --max 1 " " {})[1]' --query=(commandline --current-token)
         )
         if test $status -eq 0
             set abbreviated_commit_hash (string split --max 1 " " $selected_log_line)[1]
             set commit_hash (git rev-parse $abbreviated_commit_hash)
-            commandline --insert $commit_hash
+            commandline --current-token --replace $commit_hash
```

Comment:
I wonder if we should just allow multi-select here. Don't worry about changing it for this PR, but I'd like to hear your thoughts. It seems you use my plugin even more than me and have a better grasp on what it's good for.

Reply:
Sorry for the (really) late reply! I intended to get to this earlier but just forgot.
It's fine to keep it as-is, but I don't see any problem to always allow multiple selections. In fact I have `--multi` in my `FZF_DEFAULT_OPTS`.

Reply:
Lol how did you find this question after 2.5 months?

Reply:
It's been lying in my inbox along with many others 😅

Reply:
I see haha. Well, thanks for getting back to me. I think I'll leave it be without --multi for now.